sdk/storage/azure-storage-file/azure/storage/file/share_client.py

```diff
@@ -623,6 +624,69 @@ def list_directories_and_files( # type: ignore
         return directory.list_directories_and_files(
             name_starts_with=name_starts_with, marker=marker, timeout=timeout, **kwargs)
 
+    @staticmethod
+    def _create_permission_for_share_options(file_permission,  # type: str
+                                             **kwargs):
+        options = {
+            'share_permission': SharePermission(permission=file_permission),
+            'cls': deserialize_permission_key,
+            'timeout': kwargs.pop('timeout', None),
+        }
+        options.update(kwargs)
+        return options
+
+    @distributed_trace
+    def create_permission_for_share(self, file_permission,  # type: str
+                                    timeout=None,  # type: Optional[int]
+                                    **kwargs  # type: Any
+                                    ):
```

Comment:
This formatting is a bit strange. This could be one line:
```
def create_permission_for_share(self, file_permission, timeout=None, **kwargs):
    # type: (str, Optional[int], Any) -> str
````
However it must be over multiple lines, the formatting should look like this:
```
def create_permission_for_share(
    self,
    file_permission,  # type: str
    timeout=None,  # type: Optional[int]
    **kwargs  # type: Any
):
    # type: (...) -> str

Reply:
Let's fix all later, there are a lot of places has the same format 😀